### ISSUE TITLE
High Bloom in Day level in SSX

### ISSUE THREAD

**Issue by MasterWitcher22** [RPCS3/rpcs3#15896]
### Quick summary

The bloom in levels in the which are played during daylight is extremely high.

### Details

The bloom in levels in the which are played during daylight is extremely high  causing the track to be very bright and unconformable to play. I have given screenshot of the first tutorial level in rockies at the start of the game , the next tutorial is also very birght in the sunny areas.

### Attach a log file

[RPCS3.log](https://github.com/user-attachments/files/16493023/RPCS3.log)

### Attach capture files for visual issues

![BLUS30849_screenshot_2024_07_30_19_20_54](https://github.com/user-attachments/assets/243d3254-7211-40ec-a0f5-612890c009d4)

### System configuration

Ryzen 5 7600
Nvdia Rtx 3080ti
32gb Ram

**Comment by Megamouse** [comment]
Issues with patches applied are not allowed

**Comment by ts95** [comment]
Isn't this a problem regardless of patches being applied? I'm personally a little invested in this getting fixed since at this point it's essentially the only fault left for this game. Otherwise it looks great on rpcs3

**Comment by Siphonay** [comment]
I would also vouch for this issue to be re-opened as I can corroborate this happens without patches. This issue is documented on the wiki ass well, but with no attached issue.

**Comment by digant73** [comment]
probably still an issue, but please provide an update on that issue

**Comment by Zenko64** [comment]
I am struggling to play the game due to this issue. Happens with or without patches

**Comment by digant73** [comment]
please an update on current build. probably fixed

**Comment by dq8** [comment]
https://youtu.be/3r5Du6ce6b0?t=71
Unfortunately still an issue on the latest build. Sometimes I can't differentiate between the snow and the sky. Sometimes can't see where I'm going or what I'm doing.

**Comment by alexconsole04** [comment]
We can clearly see that.

**Comment by SlyCooperReloadCoded** [comment]
Still an issue as of 0.0.41-19565, but only on around half of the tracks. The other half look fine. Something else that happens when this issue happens is that fog becomes really distorted, like it's an object with vertex explosions, but this only happens on these incorrectly-bright tracks.

**Comment by Zenko64** [comment]
Recently the game started having more graphical glitches, Not sure if its just me. Happens after restarting or replaying a level. Then it crashes on some levels.

**Comment by dq8** [comment]
I just decided to test today on 0.0.42-20011 and seems to be fixed for me. Anyone else?

**Comment by ShadowH69** [comment]
> I just decided to test today on 0.0.42-20011 and seems to be fixed for me. Anyone else?

I will check in an hour or two.

**Comment by ShadowH69** [comment]
seems fixed to me

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/f5268f46-eb1e-45a9-b2f7-e1e505cf1442" />

**Comment by Siphonay** [comment]
Yeah, seems fixed to me too. The most affected level in my opinion was the daytime rides in the Himalayas, I just tried them and had no visibility issues at all.
